src/integration-test/java/uk/gov/ida/integrationtest/EidasUserAccountCreationIntegrationTest.java

```diff
@@ -1,86 +0,0 @@
-package uk.gov.ida.integrationtest;
```

Comment:
Has all of the eIDAS user account creation functionality been removed from the MSA in this PR or just the tests?